README.md

```diff
@@ -97,16 +95,16 @@ In your application bundle add a `plist` file named `Auth0.plist` with the follo
 <plist version="1.0">
 <dict>
   <key>ClientId</key>
-  <string>{YOUR_CLIENT_ID}</string>
+  <string>YOUR_AUTH0_CLIENT_ID</string>
```

Comment:
Is this not breaking variable replacement in samples?

Reply:
This is just a README, if you are a developer you're not going to be relying upon variable replacement. The above is done inside the `auth0.plist` inside the auth0-sample.